test/example_data.dart

```diff
@@ -804,7 +804,7 @@ InitialSnapshot initialSnapshot({
     zulipMergeBase: zulipMergeBase ?? recentZulipVersion,
     alertWords: alertWords ?? ['klaxon'],
     customProfileFields: customProfileFields ?? [],
-    emailAddressVisibility: EmailAddressVisibility.everyone,
+    emailAddressVisibility: emailAddressVisibility ?? EmailAddressVisibility.everyone,
```

Comment:
Thanks! And one more thing I missed last time. In the commit message summary line—

```
initial_snapshot: Add emailAddressVisibility override in InitialSnapshot method
```

- Keep it to 76 characters or shorter
- Let's use "test:" for the prefix, so it's clear it's purely about test code
- The method's name is `initialSnapshot`, not `InitialSnapshot`. Our convention is to import the example-data methods as `eg.`, so `eg.initialSnapshot` is a compact way to identify the method that's nice and clear that it's an example-data method.

So, how about:

```
test: Fix eg.initialSnapshot to use emailAddressVisibility override
```

Reply:
Okay, thank you. Will update that.

Reply:
I have updated the commit message